include/onnxruntime/core/platform/threadpool.h

```diff
@@ -164,6 +164,22 @@ class ThreadPool {
   void ParallelFor(std::ptrdiff_t total, const TensorOpCost& cost_per_unit,
                    const std::function<void(std::ptrdiff_t first, std::ptrdiff_t)>& fn);
 
+#ifdef _OPENMP
+  static void TryParallelFor(concurrency::ThreadPool* tp, std::ptrdiff_t total, const TensorOpCost&,
+                             const std::function<void(std::ptrdiff_t first, std::ptrdiff_t last)>& fn) {
+    std::ptrdiff_t num_threads = concurrency::ThreadPool::NumThreads(tp);
```

Comment:
what's the reason for including the function signature within the ifdef vs using ORT_UNUSED_PARAMETER for cost_per_unit if the #ifdef was just for the whole body?

Reply:
Not sure. I was following what TryBatchParallelFor did :-). Let me see if using ORT_UNUSED_PARAMETER works.